service/sign/handlers_sign.go

```diff
@@ -254,10 +255,22 @@ func Serial(w http.ResponseWriter, r *http.Request) response.ErrorResponse {
 	return response.ErrorResponse{Success: true}
 }
 
+// CleanHeader removes single quotes and leading and trailing white spaces from the header
+func CleanHeader(header string) string {
+	header = strings.Replace(header, "'", "", -1)
```

Comment:
This looks fine - there are TrimPrefix and TrimSuffix that will remove the leading and trailing characters. But I think this is fine.